user/code-climate.md

```diff
@@ -85,6 +85,20 @@ data to Code Climate requires adding the following to your .travis.yml:
     after_script:
       - vendor/bin/test-reporter
 
+### Python
+
+Python projects can use [Code Climate's Python
+reporter](https://github.com/codeclimate/python-test-reporter) to collect code coverage
+data.
+
+To set it up for your project, follow the instructions in the
+[README](https://github.com/codeclimate/python-test-reporter#codeclimate-test-reporter).
+
```

Comment:
We could delete "+Generally, you'll need to:" entirely? and just follow on from "To set it up........:"